cmd/kaniko-docker/main.go

```diff
@@ -119,6 +121,16 @@ func main() {
 			Usage:  "Cache timeout in hours. Defaults to two weeks.",
 			EnvVar: "PLUGIN_CACHE_TTL",
 		},
+		cli.StringFlag{
+			Name:   "digest-file",
+			Usage:  "Digest file Location",
+			EnvVar: "PLUGIN_DIGEST_FILE",
+		},
+		cli.StringFlag{
+			Name:   "output-file",
+			Usage:  "output file Location",
```

Comment:
location of artifact file that will be generated by the plugin. This file will include information of docker images that are uploaded by the plugin.

Reply:
done